README.rdoc

```diff
@@ -205,45 +205,32 @@ You kick butt. You've got your code reviewed and now you're ready to merge it do
 Reflow's +deliver+ requires you to have a pull request, so you'll be protected on those mornings when the coffee isn't working yet.
 We built this <b>to get in your way and make you follow the process</b>. If you don't like it, do it by hand. You already know what you're doing.
 
-You'll be presented with a prefilled commit message based on the body of your pull request which includes the text <code>Closes #XX</code> that will automatically close the associated pull request on github when deliver completes.
+You'll be presented with a prefilled commit message based on the body of your pull request which includes the text <code>Merges #XX</code> that will automatically merge the associated pull request on github when deliver completes.
```

Comment:
This wording is now misleading.  Let's change to `You'll be presented with a prefilled commit message based on the body of your pull request with references to the pull request and reviewers.`